pandas/tests/series/test_reductions.py

```diff
@@ -223,3 +223,11 @@ def test_median_with_convertible_string_raises():
     df = ser.to_frame()
     with pytest.raises(TypeError, match=msg):
         df.median()
+
+
+def test_mean_with_skipna():
+    # GH#59965 skipna=True operations don't skip NaN in FloatingArrays
+    series1 = Series({"a": 0.0, "b": 1, "c": 1}, dtype="Float64")
+    series2 = Series({"a": 0.0, "b": 2, "c": 2}, dtype="Float64")
+    result = series1 / series2
+    assert pd.notna(result.mean(skipna=True))
```

Comment:
Nit: can you assert the value here (it is a slightly stronger test). Can use `np.isclose`.